WebService/Startup.cs

```diff
@@ -76,7 +76,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             // application container, register for the "ApplicationStopped" event.
             appLifetime.ApplicationStopped.Register(() => this.ApplicationContainer.Dispose());
 
-            appLifetime.ApplicationStarted.Register(() => this.ApplicationContainer.Resolve<IRecurringTasks>().Run());
+            // appLifetime.ApplicationStarted.Register(() => this.ApplicationContainer.Resolve<IRecurringTasks>().Run());
```

Comment:
Can you add a comment as to why we are commenting out this line?

Reply:
Don't merge this change change in master. I just needed a branch with the changes for simulation.